blueprints/lib/files/lib/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Why are we adding this file? I feel like this belongs to `ember-cli-eslint`, but really we need a way for addons to hook into the base blueprints.